web-backend/src/main/java/com/statusneo/vms/util/EmailTemplateProcessor.java

```diff
@@ -0,0 +1,28 @@
+package com.statusneo.vms.util;
+
+import org.springframework.stereotype.Component;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UncheckedIOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+
+@Component
+public class EmailTemplateProcessor {
+    public String loadTemplate(String templateName, Map<String, String> placeholders) {
```

Comment:
All of this is unnecessary if we use JTE support. Spring/JTE should load it for you from standard class path location and do the rendering too.